app/Http/Controllers/Frontend/UserController.php

```diff
@@ -19,7 +19,8 @@ public function muteUser(Request $request): RedirectResponse {
                                             'user_id' => [
                                                 'required',
                                                 'exists:users,id',
-                                                Rule::notIn(auth()->user()->mutedUsers->pluck('id'))
+                                                Rule::notIn(auth()->user()->mutedUsers->pluck('id')),
+                                                Rule::notIn([auth()->user()->id]),
```

Comment:
Why is the second rule called with an array? Could both lines be merged in one rule that's called with an array?

Reply:
Because "notIn" expects an array of values that are not allowed. We can create an array and out all values in it but I think this would be more messy as these solution